src/workerd/jsg/dom-exception.h

```diff
@@ -79,11 +80,13 @@ class DOMException: public Object {
   JSG_RESOURCE_TYPE(DOMException) {
     JSG_INHERIT_INTRINSIC(v8::kErrorPrototype);
 
+    // TODO(conform): Per the spec, these should be prototype properties
+    // and not instance properties.
```

Comment:
NIT: It would be nice to link to the spec here just in case someone else wants to pick up the TODO.